kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -16,6 +16,8 @@
         <span class="middle-breadcrumb landscape" v-for="crumb in topicCrumbs">
           <a v-link="topicLink(crumb.id)">{{ crumb.title }}</a>
         </span>
+        <!-- Had to add a space or else the breadcrumbs would jump.-->
+        <span class="middle-breadcrumb landscape"> {{' ' + title}}</span>
```

Comment:
since the current title is also shown as the page title, and this last bit of breadcrumb is not clickable, might just omit it to make the breadcrumb shorter